d2l-seek-bar.js

```diff
@@ -86,19 +126,21 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-seek-bar">
 				width: calc(100% - var(--calculated-inner-knob-margin)*2);
 				height: calc(100% - var(--calculated-inner-knob-margin)*2);
 				background-color: var(--calculated-d2l-inner-knob-color);
-				border: 2px solid var(--calculated-d2l-inner-knob-color);
 				border-radius: 50%;
 				box-sizing: border-box;
+				box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.07)
 			}
 
 		</style>
 
-		<div id="sliderContainer">
+		<div id="sliderContainer" fullWidth$="{{fullWidth}}">
```

Comment:
`{{ }}` is for 2 way binding, otherwise you can use `[[ ]]`
```suggestion
		<div id="sliderContainer" fullWidth$="[[fullWidth]]">
```

Reply:
And yes you'll probably see a lot of old code doing it wrong since in the early days we didn't enforce the standard as much

Reply:
Ohhh okay, yeah I was just following what I'd seen before.